src/averageAges.js

```diff
@@ -13,13 +13,28 @@
  * @param {number} century - optional
  *
  * @return {number}
+ *
  */
+const calculateAverageAge = function(array) {
+  const length = array.length;
+
+  return array.reduce((sum, age) => sum + age, 0) / length;
```

Comment:
useless variale actually)
```suggestion
  return array.reduce((sum, age) => sum + age, 0) / array.length;
```